cypress/e2e/projectCard.cy.js

```diff
@@ -0,0 +1,75 @@
+// testing suite for project card component 
+// testing functionality of favorite, open project, and delete button
+// added it.only to line 11, to prevent accidentally deleting projects
```

Comment:
it.only will only run the test that has it.only instead of running all the tests within the test file. Cypress automatically re-runs the test with any changes including typing a "space" so I used it.only on something I was okay re-rerunning until i was ready to run the entire test file. 
I then kept it there in case I accidentally open the spec file and did not want to delete exisiting projects